main.py

```diff
@@ -2,6 +2,67 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
+
+def crud_for_genres() -> QuerySet:
+    # Create a new genres in table:
+    genres = ["Western", "Action", "Dramma"]
```

Comment:
There is a typo in the genre name 'Dramma'. It should be 'Drama'. This is corrected later in the update operation, but it's better to fix it here as well to maintain consistency.